.gitignore

```diff
@@ -3,4 +3,4 @@ dist
 extension/background.js
 extension/content_script.js
 extension/overview/overview.js
-
+extension/options/options.js
```

Comment:
Hey, I've run git filter-branch to remove all traces of `options.js` from the commit history. Commit `50a2ffB` no longer includes the file.